src/expressions/plyql.pegjs

```diff
@@ -233,39 +226,70 @@ function constructQuery(distinct, columns, from, where, groupBys, having, orderB
     }
   }
 
+  return groupBys;
+}
+
+function staticColumn(column) {
+  return column.expression.getFreeReferences().length === 0;
+}
+
+function constructQuery(distinct, columns, from, where, groupBys, having, orderBy, limit) {
+  if (!columns) error('Can not have empty column list');
+
   var query = null;
-  if (!groupBys) {
-    query = from;
-
-    if (Array.isArray(columns)) {
-      var attributes = [];
-      for (var i = 0; i < columns.length; i++) {
-        var column = columns[i];
-        query = query.performAction(column);
-        attributes.push(column.name);
-      }
-      query = query.select.apply(query, attributes);
+
+  if (!distinct && Array.isArray(columns) && !from && !where && !groupBys && columns.every(staticColumn)) {
+    // This is a SELECT 1+1; type query
```

Comment:
`select 1+1 from wikipedia` would be covered buy the else case. select 1+1 is the special case where we do not want to add any data definitions